src/_gettsim/parameters/unterhaltsvors.yaml

```diff
@@ -30,25 +30,91 @@ altersgrenze_mindesteinkommen:
+      können seit Juli 2017 auch Kinder bis zur Vollendung des 18. Lebensjahres
+      Unterhaltsvorschuss bekommen, wenn das Elternteil ein Mindesteinkommen hat.
     en: >-
       Children under the age of 12 living with a single parent are entitled to alimony
-      payments. In addition, children up to the age of 18 receive advance on alimony
-      payments if the parent has income beyond a minimal threshold.
+      payments. In addition, since July 2017, children up to the age of 18 receive
+      advance on alimony payments if the parent has income beyond a minimal threshold.
   reference: § 1 Abs. 1, 1a UhVorschG
-  2017-01-01:
-    1: 6
-    2: 12
-    3: 18
+  2008-01-01:
+    1:
+      min_alter: 0
+      max_alter: 5
+    2:
+      min_alter: 6
+      max_alter: 11
+    reference: G. v. 21.12.2007 BGBl. I S. 3194
+faktor_jüngste_altersgruppe:
+  name:
+    de: >-
+      Faktor mit dem das sächliche Existenzminimum multipliziert wird um den
+      Unterhaltsvorschuss für Kinder der jüngsten Altersgruppe zu berechnen.
+    en: >-
+      Factor by which the sächliche Existenzminimum is multiplied to calculate
+      the advance child alimony for children of the youngest age group.
+  2009-01-01:
+    scalar: 0.87
+    reference: § 1612a Abs. 1 BGB
+unterhaltsvors_anwendungsvors:
+  name:
+    de: Unterhaltsvorschuss für Kinder entsprechend Anwendungsvorschriften
+    en: Advance Child Alimony according to application regulations
+  description:
+    de: >-
+      Der Unterhaltsvorschuss wird 2009 bis 2015 prinzipiell berechnet basierend auf den
+      Parametern Kinderfreibetrag für das sächliche Existenzminimum und dem Kindergeld
+      für das erste Kind. Wird das Kindergeld unterjährig geändert oder der
+      Kinderfreibetrag unterjährig rückwirkend geändert, gelten abweichende
+      Anwendungsvorschriften, die von unveränderten Werten für Kindergeld bzw.
+      Kinderfreibetrag ausgehen können. Hier werden die entsprechenden Werte für den
+      Unterhaltsvorschuss (d.h. nach Abzug des unterstellten Kindergeldes) verwendet.
+    en: >-
+      The advance child alimony is calculated in principle from 2009 to 2015 based on
+      the parameters parameters 'child allowance for the material minimum subsistence
+      level' (Kinderfreibetrag für das sächliche Existenzminimum) and Kindergeld for the
+      first child. If the Kindergeld is changed during the year or the or the child
+      allowance is changed retroactively during the year, deviating application rules
+      apply, which can be based on unchanged values for Kindergeld or child child
+      allowance can be assumed. Here, the corresponding values for the advance child
+      alimony (i.e. after deduction of the assumed Kindergeld) are used.
+  unit: Euro
+  reference_period: Month
+  2015-01-01:
+    1: 133
+    2: 180
+    reference: null
+    note: >-
+      Der Mindesunterhalt errechnet sich aus einem Kinderfreibetrag für das sächliche
```

Comment:
I don't quite get how move from the description to the numbers. Can we add a sentence making that explicit?